### ISSUE TITLE
problem with www. and non www. addresses

### ISSUE THREAD

**Issue by Danilopereira82** [StevenBlack/hosts#1168]
i noticed something

4chan.org is blocked
but www.4chan.org is not

makes me think
how many other sites in the list are possible to access by simply using www. or not using it
maybe this is a big problem

**Comment by StevenBlack** [comment]
We've discussed this many times before, @Danilopereira82.

For starters, there's nothing magical about the `www` subdomain.  This is just another subdomain that's sometimes used by archaic convention, but isn't proscribed or standard.  

We don't block `4chan.org` in the base hosts file.  We do, however, block subdomains such as 
`engine.4chan-ads.org` and `static.4chan-ads.org` in the base hosts file.

The domain `boards.4chan.org` is where code and other things are found, and that is blocked by the porn extension.  The `www.boards.4chan.org` does not exist, so there's actually no problem there.

As for `www.4chan.org`, that's not pornographic.  The adult links off that domain (`boards.4chan.org`) are blocked.  Some links are to `boards.4channel.org`, but those are not problematic.

In short, I don't see a problem here.

Closing.
